filebeat/docs/inputs/input-udp.asciidoc

```diff
@@ -0,0 +1,42 @@
+:type: udp
+
+[id="{beatname_lc}-input-{type}"]
+=== UDP input
+
+++++
+<titleabbrev>UDP</titleabbrev>
+++++
+
+Use the `udp` input to read events over UDP.
+
+Example configuration:
+
+["source","yaml",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- type: udp
+  max_message_size: 10240
+----
+
+
+==== Configuration options
+
+The `udp` input supports the following configuration options plus the
+<<{beatname_lc}-input-{type}-common-options>> described later.
+
+//REVIEWERS: PH: What other options need to be documented here? I don't see host
+//and port listed in the reference.yml. Are the host and port separate options?
+//What guidance should we provide to help users set these options? Config ex
+//above should include this as an eample, too.
```

Comment:
@dedemorton Its currently a [single option](https://github.com/elastic/beats/blob/master/filebeat/input/udp/config.go#L13). 

It should be:

```yaml
filebeat.inputs:
- type: udp
  max_message_size: 10240
  host: "localhost:8080"
```

The default value is "localhost:8080", To be honest, I think we should not provide a default for it and enforce user to set it, so making sure the example provide the host will help us move forward.

FYI, I would like to make it mandatory for 7.0